frontend/src/stores/ParticipantStore.js

```diff
@@ -383,7 +383,7 @@ export class ParticipantStore {
       }
       this.setRouteToQueue(true)
     } catch (error) {
-      const snackbarError = handleSnackbarError(error.message)
```

Comment:
These should stay as they are. Error is an object, and whatever you pass to the constructor is available as value of a `message` key. `new Error(<accessable at key 'message'>)` its stored as a string, which is why in the error/index.js file we are parsing the integer from the parameter